### PR TITLE
bug fixes

### PR DESCRIPTION
**Pull request by bartr**
## Purpose of PR

- renamed to ~/.ssh/fluent-bit.key for consistency
- move VM cli to fleet-vm repo
- updated template
- updated make
- renamed akdc bin to flt
- moved inner-loop cli to cse-labs/kubernetes-in-codespaces repo

# Type of PR

- [ ] Documentation changes
- [x] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

### CODE REVIEW

File: bin/.akdc/commands/local-check/webv [hunk 1]
```diff
@@ -1,9 +0,0 @@
-#!/bin/bash
-
-#name: webv
-#short: Check WebV status on the local cluster
-
-# change to this directory
-#cd "$(dirname "${BASH_SOURCE[0]}")" || exit
-
-kubectl exec -it jumpbox -- http webv.imdb.svc.cluster.local:8080/version
```
Comment (kev-ms) [inline bin/.akdc/commands/local-check/webv:9]: This is being referenced from webv app command `kic apps webv`: https://github.com/retaildevcrews/akdc/blob/56b43a3931ac58ade366009029d7370daee6acb7/bin/.akdc/commands/dev/webv#L29

**Review by kev-ms — APPROVED** [review]

File: fleet/setup/k3d.yaml [hunk 2]
```diff
@@ -9,28 +9,13 @@ volumes:
   - volume: /prometheus:/prometheus
```
Comment (bartr) [inline fleet/setup/k3d.yaml:9]: should we remove the Prometheus volume too? I think we will eventually deploy Prom to the fleet, so lean towards leaving it.

I don't think we'll deploy Grafana to the fleet, but maybe?  Should we leave it?

File: fleet/setup/k3d.yaml [hunk 3]
```diff
@@ -9,28 +9,13 @@ volumes:
...
-    nodeFilters:
-      - server[0]
 ports:
   - port: 443:443
     nodeFilters:
       - loadbalancer
   - port: 80:80
     nodeFilters:
       - loadbalancer
-  - port: 30000:30000
```
Comment (bartr) [inline fleet/setup/k3d.yaml:22]: we use traefik instead of node ports - removed for clarity / simplicity

File: src/kic/Makefile [hunk 4]
```diff
@@ -6,14 +6,15 @@ build :
 	@# build
 	@gofmt -s -w .
 	@go build -o ../../bin/akdc main.go
+	@go build -o ../../../inner-loop/bin/kic main.go
```
Comment (bartr) [inline src/kic/Makefile:9]: ../../../inner-loop is the cse-labs/kubernetes-in-codespaces repo

build kic and add to path

File: .devcontainer/on-create.sh [hunk 5]
```diff
@@ -8,7 +8,7 @@ echo "$(date +'%Y-%m-%d %H:%M:%S')    on-create start" >> "$HOME/status"
 export REPO_BASE=$PWD
 export AKDC_REPO=retaildevcrews/edge-gitops
 
-export PATH="$PATH:$REPO_BASE/bin"
+export PATH="$PATH:$REPO_BASE/bin:$(dirname "$REPO_BASE")/inner-loop/bin"
```
Comment (bartr) [inline .devcontainer/on-create.sh:11]: inner-loop is the cse-labs/kubernetes-in-codespaces repo

we build kic and put in the inner-loop/bin dir so we need it in the path

File: .devcontainer/on-create.sh [hunk 6]
```diff
@@ -27,20 +27,17 @@ cp src/_* "$HOME/.oh-my-zsh/completions"
...
     # create aliases
     echo "alias mk='cd $REPO_BASE/src/kic && make build; cd \$OLDPWD'"
-    echo "alias kic='akdc local'"
     echo "alias flt='akdc fleet'"
 
     echo "export REPO_BASE=$PWD"
     echo "export AKDC_REPO=retaildevcrews/edge-gitops"
     echo "export AKDC_SSL=cseretail.com"
     echo "export AKDC_GITOPS=true"
-    echo "export KIC_PATH=/workspaces/akdc/bin"
```
Comment (bartr) [inline .devcontainer/on-create.sh:42]: moved these to the makefile so that kic will work

File: .devcontainer/on-create.sh [hunk 7]
```diff
@@ -87,20 +84,20 @@ git clone https://github.com/cse-labs/imdb-app
 git clone https://github.com/cse-labs/kubernetes-in-codespaces inner-loop
 git clone https://github.com/retaildevcrews/edge-gitops
 git clone https://github.com/retaildevcrews/red-gitops
-cd "$OLDPWD" || exit
+cd "$REPO_BASE" || exit
```
Comment (bartr) [inline .devcontainer/on-create.sh:87]: this seems less error prone - a second `cd` won't break this

File: .devcontainer/on-create.sh [hunk 8]
```diff
@@ -87,20 +84,20 @@ git clone https://github.com/cse-labs/imdb-app
...
 
-echo "creating k3d cluster"
-pushd ../inner-loop || exit
+echo "generating kic completion"
+kic completion zsh > "$HOME/.oh-my-zsh/completions/_kic"
 
-akdc local cluster rebuild
-popd || exit
+echo "creating k3d cluster"
```
Comment (bartr) [inline .devcontainer/on-create.sh:97]: we automatically get any updates to cse-labs/kubernetes-in-codespaces

File: .devcontainer/post-create.sh [hunk 9]
```diff
@@ -17,7 +17,7 @@ echo "$ID_RSA_PUB" | base64 -d > "$HOME/.ssh/id_rsa.pub"
 
 # save keys
 echo "$AKDC_MI" > "$HOME/.ssh/mi.key"
-echo "$LOKI_URL" > "$HOME/.ssh/loki.key"
+echo "$LOKI_URL" > "$HOME/.ssh/fluent-bit.key"
```
Comment (bartr) [inline .devcontainer/post-create.sh:20]: standardize with VM key name

File: .devcontainer/post-create.sh [hunk 10]
```diff
@@ -17,7 +17,7 @@ echo "$ID_RSA_PUB" | base64 -d > "$HOME/.ssh/id_rsa.pub"
...
 echo "$PROMETHEUS_KEY" > "$HOME/.ssh/prometheus.key"
 
 # set file mode
```
Comment (bartr) [inline .devcontainer/post-create.sh:23]: all of the deleted command files are in cse-labs/kubernetes-in-codespaces/bin/.kic/commands

this lets us maintain a single version

File: fleet/setup/k3d.yaml [hunk 11]
```diff
@@ -9,28 +9,13 @@ volumes:
...
-      - server[0]
-  - port: 31080:31080
-    nodeFilters:
-      - server[0]
-  - port: 32000:32000
-    nodeFilters:
-      - server[0]
 
 options:
   k3d:
```
Comment (bartr) [inline fleet/setup/k3d.yaml:21]: the entire fleet directory is now redundant as it exists in red-gitops and edge-gitops

I think we should remove from here to simplify maintenance

Should we continue to keep in *-gitops or should we create a "fleet-cli" repo and clone that to the VMs as part of setup?

edge-gitops was out of sync with red-gitops and also with akdc/fleet. One source of truth (repo) would fix that. But, it's another repo.

Comment (bartr) [reply fleet/setup/k3d.yaml:21]: we can also remove the commands/vm directory as it is redundant

when we do that, we can drop the `alias flt=akdc fleet` and just compile to flt

then you would have the `flt` and `kic` commands in akdc and a different kic command in the VMs.

that seems cleaner

**Pull request merged**